.evergreen/prepare-crypt-shared-lib.sh

```diff
@@ -12,7 +12,6 @@ get_mongodb_download_url_for "$DISTRO" "$MONGODB_VERSION"
 # get_mongodb_download_url_for defines $MONGO_CRYPT_SHARED_DOWNLOAD_URL and $EXTRACT.
 if [ -z "$MONGO_CRYPT_SHARED_DOWNLOAD_URL" ]; then
   echo "There is no crypt_shared library for distro='$DISTRO' and version='$MONGODB_VERSION'".
-  exit 1
```

Comment:
Not exiting here allows us to run the same script in evergreen for all csfle tests, so it continues running when server is less than 6.0 and the env var doesn't get set.